### ISSUE TITLE
Dockerized Setup for Sonar

### ISSUE THREAD

**Issue by hardlyhuman** [aidecentralized/sonar#12]
I propose adding a Dockerized setup for the Sonar project. This would streamline the development and deployment processes, ensuring a consistent environment across all stages and also ease the process of setting up for the new developers.

I can work on this after issue #7

**Comment by tremblerz** [comment]
That is a great point! Right now a few people ran into the problem of ubuntu vs mac. I recommend working on this after #7

**Comment by hardlyhuman** [comment]
Yes, I am one among those. I can start working on this

**Comment by tremblerz** [comment]
Yes, I don't know how triton entered as a topology thanks for removing it. Go ahead with the PR.
